apps/gauzy/src/app/pages/dashboard/custom-dashboard/dashboard-tab-strip.component.scss

```
@@ -0,0 +1,171 @@
+	align-items: center;
+	min-width: 0;
+
+	&__list {
+		display: flex;
+		align-items: center;
+		gap: 0.375rem;
+		flex: 1 1 auto;
+		min-width: 0;
+		padding: 0.25rem 0;
+		overflow-x: auto;
+		// The strip scrolls horizontally without showing a scrollbar
```

Stylelint (17.14.0)

[error] Unknown word The (CssSyntaxError)

(CssSyntaxError)